code/game/machinery/doors/poddoor.dm

```diff
@@ -132,10 +137,10 @@
 	switch(animation)
 		if("opening")
 			flick("opening", src)
-			playsound(src, 'sound/machines/blastdoor.ogg', 30, TRUE)
+			playsound(src, opening_sound, 50, TRUE)
 		if("closing")
 			flick("closing", src)
-			playsound(src, 'sound/machines/blastdoor.ogg', 30, TRUE)
+			playsound(src, opening_sound, 50, TRUE)
```

Comment:
these are both `opening_sound`?

Reply:
oh i just noticed the description where it says "It having two different sound effects is just to show what they both sound like; in-game the shutters just have the new sound for opening and closing." this is still really confusing as a var name anyways, should be more aptly named